services/sequelize/recovery_method.js

```diff
@@ -0,0 +1,169 @@
+const { Op } = require('sequelize');
+
+const models = require('../../models');
+
+const { Account } = models;
+
+const SequelizeRecoveryMethods = {
+    cleanRecoveryMethod(recoveryMethod) {
+        if (!recoveryMethod) {
+            return null;
+        }
+
+        const {
+            kind,
+            detail,
+            publicKey,
+            requestId,
+            securityCode,
+            createdAt,
+            updatedAt,
+        } = recoveryMethod.toJSON();
+
+        return {
+            kind,
+            detail,
+            publicKey,
+            requestId: parseInt(requestId, 10),
+            securityCode,
+            createdAt,
+            updatedAt,
+        };
+    },
+
+    async createRecoveryMethod({ accountId, detail, kind, publicKey, requestId, securityCode }) {
+        const account = await Account.findOne({ where: { accountId } });
+        const recoveryMethod = await account.createRecoveryMethod({
+            kind,
+            ...(detail && { detail }),
+            ...(publicKey && { publicKey }),
+            ...(requestId && { requestId }),
```

Comment:
Although we store this as a string in our SQL DB currently, it is a number in the smart contract so we can assume it will be POST'd to us as a number.  Given that fact, if `requestId` is 0, this will be falsy and not included in the payload, but that is a valid index for `requestId` and should be posted as an attribute here AFAICT